Makefile

```diff
@@ -285,6 +323,14 @@ docker-buildx-release:
 		-t $(REPOSITORY):$(VERSION) \
 		. --push
 
+docker-buildx-crds-release: build-crds
+	@if ! DOCKER_CLI_EXPERIMENTAL=enabled docker buildx ls | grep -q container-builder; then\
+		DOCKER_CLI_EXPERIMENTAL=enabled docker buildx create --name container-builder --use;\
+	fi
+	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --build-arg LDFLAGS=${LDFLAGS} --platform "linux/amd64,linux/arm64,linux/arm/v7" \
```

Comment:
is `buildx` still experimental? Should we also support docker build?

Reply:
docker build update was added here: https://github.com/open-policy-agent/gatekeeper/pull/1485/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R281

Reply:
ack, thanks